CMakeLists.txt

```diff
@@ -15,6 +15,17 @@ else()
   set(BUILD_DEPS "/usr/local/osquery")
 endif()
 
+if(WIN32)
+  set(WINDOWS_DEP_DIR "$ENV{ChocolateyInstall}/lib")
+  string(REGEX REPLACE "\\\\" "/" WINDOWS_DEP_DIR ${WINDOWS_DEP_DIR})
+  list(APPEND WINDOWS_DEP_LIST ${BUILD_DEPS})
```

Comment:
Adding `BUILD_DEPS` here may be redundant. Does Windows use this directory? If not then let's remove it from the list.